sportsdataverse/nfl/nfl_season_standings.py

```
@@ -0,0 +1,1247 @@
+            .otherwise(pl.col("conf_rank"))
+            .alias("conf_rank"),
+        )
+        st = st.with_columns(
+            pl.when(pl.col("__winner") == True)  # noqa: E712
+            .then(1)
+            .otherwise(pl.col("conf_rank_counter"))
+            .alias("conf_rank_counter"),
+        )
+        st = _set_label(
+            st,
+            pl.col("__winner") == True,
```

Flake8 (7.3.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)